assignment/assignment-7/index.html

```diff
@@ -5,87 +5,151 @@
-                <input type="time" name="appoinmenttime" id="time" />
-                <label for="day">Appoinment Day</label>
-                <input type="week" name="appoinmentday" id="day" />
-                <label for="month">Appoinment Month</label>
-                <input type="month" name="appoinmentmonth" id="month" />
+            <hr />
+            <div class="location">
+                <label for="urlInput">Website</label>
+                <input
+                    type="url"
+                    id="urlInput"
+                    name="website"
+                    pattern="https://.*"
+                />
             </div>
-
-            
-            
-            <button>Submit Application</button>
+            <hr />
+            <div class="feedback">
+                <label for="feedbackInput">Satisfaction Rating</label>
+                <input
+                    type="range"
+                    name="satisfactionRating"
+                    id="feedbackInput"
+                    min="1"
+                    max="99"
+                />
+                <label for="commentsInput">Comments</label>
+                <textarea
+                    name="comments"
+                    id="commentsInput"
+                    cols="20"
+                    rows="5"
+                ></textarea>
+            </div>
+            <hr />
+            <div class="schedule">
+                <label for="meetingTimeInput">Preferred Meeting Time</label>
+                <input type="time" name="preferredTime" id="meetingTimeInput" />
+                <label for="meetingDayInput">Preferred Meeting Day</label>
+                <input type="week" name="preferredDay" id="meetingDayInput" />
+                <label for="meetingMonthInput">Preferred Meeting Month</label>
+                <input
+                    type="month"
+                    name="preferredMonth"
+                    id="meetingMonthInput"
+                />
+            </div>
+            <hr />
+            <div class="search">
+                <label for="articlesInput">Search for Articles</label>
+                <input type="search" name="searchArticles" id="articlesInput" />
+            </div>
+            <hr />
+            <div class="submit-reset">
+                <input
+                    type="submit"
+                    name="submit"
+                    value="Submit My Application"
+                />
+                <input type="reset" name="reset" value="Reset My Application" />
+                <button>
+                    Sign In
+                    <img src="./sign in.png" alt="" />
+                </button>
+            </div>
+            <div class="clickable-img">
+                <label for="imgInput">Icon Image</label>
+                <input
+                    type="image"
+                    name="iconImg"
+                    id="imgInput"
+                    src="./image.png"
+                />
```

Comment:
If you can add autofocus that would be goog.

Reply:
I added. Thanks